co-authors-plus.php

```diff
@@ -385,7 +385,9 @@ public function coauthors_meta_box( $post ) {
 		?>
 
 		<div id="coauthors-edit" class="hide-if-no-js">
-			<p><?php _e( 'Click on an author to change them. Drag to change their order. Click on <strong>Remove</strong> to remove them.', 'co-authors-plus' ); ?></p>
+			<?php add_thickbox(); ?>
+			<a class="button thickbox" href="<?php echo get_bloginfo('url'); ?>/wp-admin/users.php?&page=view-guest-authorsTB_iframe=true&width=1200&height=640"><small><?php _e( 'Guest Authors', 'co-authors-plus' ); ?></small></a>
```

Comment:
Should be spaces inside parentheses for `get_bloginfo` to be consistent with WordPress PHP coding standards.

For safe output escaping, you should assemble the URL and then `esc_url` it. Also use `esc_html_e` instead of `_e`. It's safer.